### ISSUE TITLE
Junk code injected into Bootstrap and others

### ISSUE THREAD

**Issue by darth-cheney** [UnitedLexCorp/SimpleTalk#174]
@dkrasner It appears that you have some kind of plugin or extension injecting junk code into your web pages, and this has made its way into the various core serializations. I'm guessing it has something to do with Youtube?
  
Here is an example of what I mean, from [bootstrap](https://github.com/UnitedLexCorp/SimpleTalk/blob/master/js/objects/examples/bootstrap.html#L9):
```html
<script>(function addTranscript() {
  function debounce(callback, wait, immediate = false) {
    let timeout = null;

    return function () {
      const callNow = immediate && !timeout;
      const next = () => callback.apply(this, arguments);

      clearTimeout(timeout);
      timeout = setTimeout(next, wait);

      if (callNow) {
        next();
      }
    };
  }
[ ... etc ...]
```
It is also adding classes to the main body element.
  
The commit that did this is [here](https://github.com/UnitedLexCorp/SimpleTalk/commit/dd32876caa804155e9e80b725d20bb62c7f3cc2a).

**Comment by dkrasner** [comment]
@darth-cheney i am taking care of this on the `daniel-cleanup` branch

**Comment by darth-cheney** [comment]
Closed by #184
